resources/definitions/predator.def.json

```diff
@@ -6,7 +6,7 @@
         "author": "ksihota",
         "manufacturer": "Anycubic",
         "file_formats": "text/x-gcode",
-        "platform": "predator_platform.obj",
+        "platform": "predator_platform.stl",
 		"platform_texture": "predator_image.png",

```

Comment:
png image was removed, so please remove this reference to it

Reply:
Weird the image on my side is present. I will rename to force the push (I hope).